test/helpers.py

```diff
@@ -51,7 +51,12 @@ def is_ipv6_tcp_pkt(pkt):
 	return IPv6 in pkt and TCP in pkt
 
 def is_tcp_pkt(pkt):
-	return TCP in pkt
+	if TCP in pkt:
+		for option in pkt[TCP].options:
+			if option[0] == 'MSS':
+				return False
+		return True
+	return False
```

Comment:
I know it's just pytest, but failing fast would look more in line with our code.

But for a real Pythonic code:
`return TCP in pkt and 'MSS' not in [ option[0] for option in pkt[TCP].options ]`

Reply:
definitely nicer.